tests/test_api.py

```diff
@@ -74,7 +79,59 @@ def test_api_dataset_index_unindex(app, client, search_client, faker):
     assert dataset_search_after_delete_resp.json['total'] == 0
 
 
-def test_api_orgt_index_unindex(app, client, search_client, faker):
+def test_api_dataset_index_on_another_index(app, client, search_client, faker):
+    now = datetime.datetime.now().strftime('%Y-%m-%d-%H-%M')
+    index_name = f"{app.config['UDATA_INSTANCE_NAME']}-dataset-{now}"
```

Comment:
How about a different name so that we don't confuse it with existing ones. Ex: `test-dataset-{now}`